src/scripts/main.js

```diff
@@ -18,10 +18,53 @@ const food = {
   },
 };
 
+document.body
+  .appendChild(document.createElement('div'))
+  .setAttribute('id', 'tree');
```

Comment:
```suggestion
```

Reply:
if i remove this then the page will not have the element with attribute id="tree"